pkg/ccl/streamingccl/streamingest/stream_ingestion_processor.go

```diff
@@ -57,6 +57,27 @@ var minimumFlushInterval = settings.RegisterPublicDurationSettingWithExplicitUni
 	nil, /* validateFn */
 )
 
+var maxKVBufferSize = settings.RegisterByteSizeSetting(
+	settings.TenantWritable,
+	"bulkio.stream_ingestion.kv_buffer_size",
```

Comment:
I feel like I want to nit these names namely "bulkio" as a prefix and the generic "stream" as a feature identifier without mentioning replication, but I don't really have better suggestions.

I wonder if these should just be job-detail-stored parameters instead?

Reply:
I can change them all in a follow up PR. We made the top level public package `repstream`.  We really did never find good words to use at this code level.